Makefile

```diff
@@ -117,15 +115,15 @@ clean:
 coverage:
 	@scripts/coverage.sh
 
-HAS_GLIDE := $(shell command -v glide;)
+HAS_DEP := $(shell command -v dep;)
 HAS_GOX := $(shell command -v gox;)
 HAS_GIT := $(shell command -v git;)
 HAS_HG := $(shell command -v hg;)
 
 .PHONY: bootstrap
 bootstrap:
-ifndef HAS_GLIDE
-	go get -u github.com/Masterminds/glide
+ifndef HAS_DEP
+	go get -u github.com/golang/dep/cmd/dep
```

Comment:
With `dep` being under such active development, do we want this pulling straight from `HEAD`?

Reply:
That was what was recommended in their docs, but I completely agree. Let me see if I can find an alternative.

Reply:
You can pull the binary from the GH releases

Reply:
They do! It appears to be much more stable than glide on Windows.

Reply:
the README now recommends using one of the released versions rather than the tip. if there's still another place that's suggesting tip, it's only because we missed updating it 😄